app/assets/javascripts/app.js

```diff
@@ -19,28 +19,12 @@ planner.run(['$rootScope', function($rootScope){
 
 planner.config(['$stateProvider', '$urlRouterProvider', function($stateProvider, $urlRouterProvider) {
 
-  $urlRouterProvider.otherwise('/');
+  $urlRouterProvider.otherwise('');
 
   $stateProvider
```

Comment:
Does this work? if on local I go to `localhost:3000/adsfasdf`, does this redirect correctly? personally I would make this `otherwise('/students')`. I haven't used this blank syntax before.